src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/SupportControllerTest.java

```diff
@@ -111,4 +122,21 @@ public void shouldResendRPANotifications() {
 
     }
 
+    @Test
+    public void shouldTriggerUploadDocumentWhenDefendantResponseSubmittedInvoked() {
+        when(claimService.getClaimByReferenceAnonymous(eq("000CM001"))).thenReturn(Optional.of(sampleClaim));
+        controller.resendStaffNotifications("000CM001", "response-submitted", AUTHORISATION);
+        verify(defendantResponseCitizenNotificationsHandler).uploadDocument(any(DefendantResponseEvent.class));
+    }
+
+    @Test
+    public void shouldThrowExceptionIfDefendantResponseSubmittedWhenNoDefendantResponse() {
+        exceptionRule.expect(ConflictException.class);
```

Comment:
Throwing `ConflictException` when response does not exist is misleading. Prefer to use `NotFoundException` or `ResponseNotFoundException` here.

Reply:
I have not introduced this new exception it, it was already being thrown. I am ok to change it but personally I don't see any difference. Please note : there is no "ResponseNotFoundException" as you suggeting that I should create a new class ?

Reply:
ResourceNotFound should be thrown only when a Case is missing in REST, the Resource you are referring. Here what is happening is we are trying to make an action on a resource where the state is conflicting what action you are doing. So ConflictException is correct. 

If we are trying to get /case/response when a response is not there, then we throw ResourceNotFound 404